tests/letstest/scripts/test_leauto_upgrades.sh

```diff
@@ -64,6 +64,38 @@ iQIDAQAB
 -----END PUBLIC KEY-----
 "
 
+if [ $(python -V 2>&1 | cut -d" " -f 2 | cut -d. -f1,2 | sed 's/\.//') -eq 26 ]; then
```

Comment:
Note for posterity that I don't actually want to see changed: this test is only correct in practice, because we only actually upgrade to python3 when we're on redhat based systems, whereas this test checks on any system.